code/game/objects/items/devices/megaphone.dm

```diff
@@ -32,6 +33,10 @@
 		if(H && H.mind && H.mind.miming)
 			to_chat(user, "<span class='warning'>Your vow of silence prevents you from speaking.</span>")
 			return
+		if(H.mind)
+			span = H.mind.speech_span
+		if((COMIC in H.mutations) || (locate(/obj/item/organ/internal/cyberimp/brain/clown_voice) in H.internal_organs))
```

Comment:
instead of the `locate in` check, you should use: `H.get_int_organ(/obj/item/organ/internal/cyberimp/brain/clown_voice)`